src/main/js/components/subcomponents/MenuItem.js

```diff
@@ -76,37 +104,76 @@ class ManagerMenuItemList extends React.Component {
                     {menuItems}
                     </tbody>
                 </table>
+                <div>
+                    {navLinks}
+                </div>
             </div>
         )
     }
 
 }
 
+
 class ManagerMenuItem extends React.Component {
 
     constructor(props) {
         super(props);
-        this.state = {tags: []};
+        this.state = {menuItemTags: []};
         this.handleDelete = this.handleDelete.bind(this);
+        this.requestTags = this.requestTags.bind(this);
     }
 
     handleDelete() {
-        this.props.onDelete(this.props.menuItem);
+        this.props.onDelete(this.props.menuItem, 'menuItems');
+    }
+
+    requestTags() {
+        fetch(this.props.menuItem.entity._links.tags.href, {method: 'GET', headers: {'Content-Type': 'application/json'}})
+            .then(
+                response => {
+                    if (response.status !== 200) {
+                        console.log('Looks like there was a problem. Status Code: ' +
+                            response.status);
+                        return;
+                    }
+
+                    // Examine the text in the response
+                    response.json().then((data) => {
+                        console.log(data._embedded.tags);
```

Comment:
Was this meant to be temporary for debugging?

Reply:
Yeah it was, and also on the mobile thing, as you suggest I don't think it's worth looking into - too low priority,